### PR TITLE
Additives aleene

### PR DESCRIPTION
**Pull request by aleene**
New translations for additives. Started to remove the additives from the ingredients taxonomy.

### CODE REVIEW

**Comment by stephanegigandet** [comment]
Additives is a special taxonomy with special handling in the code, we need to keep this ordering for all entries and all languages.

en:E999, Main name of additive, Synonym1, Synonym2

There is no need to list variations of the e-number like E-282, E 282, E282, INS282, INS-282, INS 282, SIN282, SIN-282, SIN 282, they are generated automatically.

**Comment by aleene** [comment]
Mmm code exceptions.

**Comment by aleene** [comment]
I'll correct it.

**Comment by aleene** [comment]
This is now repaired.

File: taxonomies/additives.txt [hunk 1]
```diff
@@ -4305,35 +4305,53 @@ efsa_evaluation:en: Scientific Opinion on the re-evaluation of propionic acid (E
...
-it:E282, Propionato di calcio
+fa:E282, کلسیم پروپانوات
+fi:E282, Kalsiumpropionaatti
+fr:E282, propionate de calcium
+hu:E282, kalcium-propionat
+it:E282, propionato di calcio
 lt:E282, Kalcio propionatas
 lv:E282, Kalcija propionāts
 mt:E282, Propjonat tal-kalċju
-nl:E282, Calciumpropionaat, Calciumpropanoaat
```
Comment (stephanegigandet) [inline taxonomies/additives.txt]: Is removing Calciumpropanoaat intended? Same for German

Comment (aleene) [reply taxonomies/additives.txt]: Yes, it is not used on products

**Pull request merged**